### DISCUSSION TITLE
[A-L] (2023/24) Foglio 11 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#331] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-23 103937](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8656f0a8-3af4-4e51-8752-72f905c0208b)

**Comment by cornflexxx** [comment]
Per ipotesi sappiamo che $v_1,...,v_k$ sono linearmente indipendenti, quindi presi $\lambda_1,...,\lambda_k\in\mathbb{R}$ con $\lambda_i\neq 0\quad\forall$  $i$, la combinazione lineare con i $\lambda_i$ è sempre diversa da $0$:
$$\lambda_1v_1 +...+\lambda_kv_k\neq 0\qquad (I)$$

Prendiamo ora $w_j=c_jv_j\$ con $j=1,...,k$, sappiamo che $c_j\in \mathbb{R}$ ,  $c_j\neq 0$  $\forall j$, riscriviamo la combinazione dei $w$ con i $\lambda$ come nella $(I)$:
$$\lambda_1w_1+...+\lambda_kw_k\neq 0$$
Sappiamo che $w_j=c_jv_j$ , quindi possiamo usare la proprietà associativa del prodotto e riscriverla nel seguente modo:
$$(\lambda_1c_1)v_1+...+(\lambda_kc_k)v_k\neq 0$$
Chiamo $\zeta_j=\lambda_jc_j$ , $\zeta_j\in\mathbb{R}$ ($\mathbb{R}$ chiuso rispetto al prodotto), $\zeta_j\neq 0$ in quanto prodotto di numeri diversi da 0. Riscriviamo sostituendo $\zeta$:
$$\lambda_1w_1+...+\lambda_kw_k=\zeta_1v_1+...+\zeta_kv_k\neq 0\qquad (II)$$

Data l'indipendenza per ipotesi l'indipendenza di $v_1,...,v_k$ allora  la $(II)$ è dimostrata.

**Comment by daromacarlo** [comment]
![Immagine 2023-11-26 192224](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/6e9647c6-c685-425f-8b5f-d913e732a4b4)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/06706e59-2794-4408-9d36-f445add61429)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/ca30712b-7e84-44fb-8ac8-094e8ae339bf)
